docs/customization.md

```diff
@@ -110,13 +110,13 @@ And the correspondent example for a yaml json file:
 commitizen:
   name: cz_customize
   customize:
-    message_template: "{{change_type}}:{% if show_message %} {{message}}{% endif %}"
+    message_template: '{{change_type}}:{% if show_message %} {{message}}{% endif %}'
```

Comment:
May I know why do we want to change from " to ' here?

Reply:
Uniformization and cleanups, updated against latest changes